api/v1alpha1/k6_types.go

```diff
@@ -91,6 +93,15 @@ type TestRunSpec struct {
 	Token     string `json:"token,omitempty"`     // PLZ reserved field (for now)
 }
 
+func (k6 TestRunSpec) isPaused() bool {
+	if k6.Paused == "" {
+		return true
+	}
+
+	paused, _ := strconv.ParseBool(k6.Paused)
+	return paused
+}
```

Comment:
There is a weird behavior here, where if what the string passed by the user in is not a boolean, it's interpreted as `false` by `ParseBool`. I'm just copying the existing behavior from where I extracted this logic from, but let me know if we should instead return `true`, our default if uninformed.

 We could avoid some of these shenanigans with #455 :-)